lib/plugin.js

```diff
@@ -75,10 +75,32 @@ exports.register = function (server, opts, next) {
         patch: patch,
         delete: del
     })
-    
+
     server.ext('onPostStop', (server, next) => {
         adapter.disconnect(next)
     })
+
+
+    server.ext('onPreResponse', (request, reply) => {
+        const response = request.response
+        if (response.isBoom) {
+            response.output = response.output || {}
+            response.output.headers = response.output.headers || {}
+            response.output.headers['content-type'] = 'application/vnd.api+json'
```

Comment:
Maybe. I'm not sure if there is a case where we don't want to return 'application/vnd.api+json'. If absolutely everything should be 'application/vnd.api+json' then it would make sense to set that once globally, and perhaps override the new default if and when necessary.